static/js/server_events_dispatch.js

```diff
@@ -352,6 +352,9 @@ export function dispatch_normal_event(event) {
                         }
                         settings_org.populate_realm_domains(page_params.realm_domains);
                         break;
+                    default:
+                        blueslip.error("realm_domains event called without any matching event op");
```

Comment:
This is good but we should probably use the error string style from daa9bbba613b55480e79673dc97f05f79ab0d3cb -- "Unexpected" rather than "without any matching", and including the value we see in the error message, since that's part of what one will need to debug these.

I appreciate the pattern of adding default blocks everywhere to help catch bugs -- as you can see we only just added that for the backend side of this.